query/language.peg

```diff
@@ -105,7 +105,7 @@ expression_function  <-
     p.addStringLiteral(unescapeLiteral(buffer[begin:end]))
   }
   PAREN_OPEN
-    expressionList (__ groupByClause)?
+    expressionList (__ groupByClause / { p.addGroupBy() } )
```

Comment:
I can! the reason why I've done this is because I wanted to make `groupByClause` to be self-containing. i.e. don't depend anything on the stack for it.

it also should only apply to the last "argument" (not really an argument, but something that gets passed along with everything).